installer/pkg/components/kubestate-metrics/deployment.go

```diff
@@ -0,0 +1,112 @@
+package kubestateMetrics
+
+import (
+	"fmt"
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	"github.com/gitpod-io/observability/installer/pkg/common"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func objectPointer[T any](o T) *T {
+	return &o
+}
```

Comment:
Any special reason to not use k8s.io/utils/pointer instead?  🤔

Reply:
Not really, other than the above being better 😅 Let the compiler do the work :)